packages/react-shopify-app-route-propagator/README.md

```diff
@@ -100,3 +100,13 @@ export default function() {
   );
 }
 ```
+
+### Troubleshooting
+
+#### `react-router` typescript issues
+
+`withRouter` has a strict type definition for the component it wraps. The component has to implement the entire `withRouter` interface, rather than a partial implementation. If you're getting a type error on the `ShopifyRoutePropagator` component, you can define it as `any`.
```

Comment:
> has to implement the entire `withRouter` interface
I think a more accurate version would be `is expected to require the entire WithRouterProps prop definition`

> you can define it as `any`

Technically what we have to do here is casting, so we can say something like 'You can cast it to any when passing it in to withRouter`